sigstore/verify/models.py

```diff
@@ -171,8 +186,6 @@ class VerificationMaterials:
     certificate: Certificate
     """
     The certificate that attests to and contains the public signing key.
```

Comment:
Can revert this TODO removal, had it in place when I was still considering certificate chains.